app/src/main/java/com/keylesspalace/tusky/util/FontScaleContextWrapper.kt

```diff
@@ -0,0 +1,43 @@
+package com.keylesspalace.tusky.util
+
+import android.content.Context
+import android.content.ContextWrapper
+
+class FontScaleContextWrapper(base: Context?) : ContextWrapper(base) {
```

Comment:
This class is not actually doing anything, could simplify by just having a top level function `wrapWithFontScaling` that just returns a `ContextWrapper`. 
(Also I don't think the `[base](base: Context?)` argument can be nullable.)